### PR TITLE
Add game logic #7

### LINKED ISSUE #7: add game logic

### PR DESCRIPTION
**Pull request by Ameer2000Mzori**

### CODE REVIEW

**Review by Ameer2000Mzori — COMMENTED** [review]
good job i like this

File: src/index.html [hunk 1]
```diff
@@ -16,8 +16,27 @@
...
+            <div>0</div>
+          </div>
+        </div>
+
+        <div class="header-Right">
+          <div>Player 2</div>
+          <div>56</div>
+          <div>
+            <div>Current</div>
+            <div>0</div>
```
Comment (Ameer2000Mzori) [inline src/index.html]: good job

File: src/index.html [hunk 2]
```diff
@@ -11,13 +11,100 @@
...
+            <div
+              class="player-scores-2 max-[810px]:text-[40px] max-sm:text-sm player-1-current-score"
+            >
+              0
+            </div>
+          </div>
+        </div>
+        <div id="header-Right" class="players-container">
+          <div>
+            <div
```
Comment (tuwafula) [inline src/index.html:57]: Everything looks great

File: src/script.js [hunk 3]
```diff
@@ -1 +1,123 @@
...
+const playerTwOCurrentScore = document.getElementsByClassName(
+  'player-2-current-score'
+)[0]
+
+// scores elements all time
+const playerOneScoresAllTime = document.getElementsByClassName(
+  'player-One-Scores-All-Time'
+)[0]
+const playerTwoScoresAllTime = document.getElementsByClassName(
+  'player-Two-Scores-All-Time'
```
Comment (tuwafula) [inline src/script.js:26]: Everything looks great

**Review by tuwafula — COMMENTED** [review]
Good job

**Pull request merged**